src/app/public/modules/text-expand-repeater/fixtures/text-expand-repeater.component.fixture.html

```diff
@@ -1 +1,9 @@
 <sky-text-expand-repeater [data]="data" [maxItems]="numItems"></sky-text-expand-repeater>
+
+<sky-text-expand-repeater [data]="data2" [maxItems]="numItems">
```

Comment:
The unit tests are now failing because we need to explicitly set the template's name:
```
<sky-text-expand-repeater [itemTemplate]="myTemplate">
```

Reply:
This has been fixed